src/hooks/useCounter.ts

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+export const useCounter = (
+	initialRating: number
+): {
+	counter: number;
+	increment: (c: number) => void;
+	decrement: (c: number) => void;
+} => {
+	const [counter, setCounter] = useState<number>(initialRating);
+
+	const increment = (c: number) => {
+		if (c === 5) {
```

Comment:
can we make constants for 5, 0.5, and 0.5? something like MAX_COUNT, MIN_COUNT, and INCREMENT_VAL?

Reply:
Yes sir!